src/MissedThrowCheckbox.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default Props => {

```

Comment:
Why is this called Props? 

Also, try to keep to a single pattern when it comes to exporting. Personally I prefer to define the const here and export default at the bottom.

Reply:
Also, where are you getting the props in the component from?

I would go for something like:

```suggestion
const MissedThrowCheckbox = ({...destructured props here} )  => (
    .....
)
```